python/raydp/tests/test_spark_cluster.py

```diff
@@ -61,6 +61,12 @@ def test_spark_driver_and_executor_hostname(spark_on_ray_small):
     driver_bind_address = conf.get("spark.driver.bindAddress")
     assert node_ip_address == driver_bind_address
 
+def test_ray_dataset_from_and_to_spark(spark_on_ray_small):
```

Comment:
Can you also add a test by creating a Ray Dataset directly and then convert to Spark dataframe?